src/calibre/utils/podofo/outline.cpp

```diff
@@ -13,7 +13,7 @@ using namespace pdf;
 static void
 dealloc(PDFOutlineItem* self)
 {
-    self->ob_type->tp_free((PyObject*)self);
+    py_TYPE(self)->tp_free((PyObject*)self);
```

Comment:
you have a tiny typo here, but otherwise LGTM.

Reply:
Oops.